site/pmguides/src/websockets_python/websockets_python.md

```diff
@@ -0,0 +1,182 @@
+author: Greg Bulmash
+id: websockets_python
+summary: Set up a WebSockets server in Python
+categories: Getting-Started
+environments: web
+status: Published
+feedback link: https://github.com/loopDelicious/pmquickstarts
+tags: Getting Started, Developer
+
+# Set up a WebSockets server in Python
+
+<!-- ------------------------ -->
+
+## Overview
+
+Duration: 1
+
+WebSockets are a way to create a persistent connection between an app or web app and a server. This can help you get as close as possible to real-time for constantly updating data like a stock ticker or text chat.
+
+The [WebSocket protocol](https://www.rfc-editor.org/rfc/rfc6455) defines how this is done. The connection is both persistent and low-latency, so communication can be initiated by the client or the server, instead of the client request / server response communication model many developers associate with the web.
+
+In this tutorial, you'll create a simple WebSocket server in Python, and use Postman to send and receive messages across the WebSocket connection.
+
+
+### Prerequisites
+
+- [Python 3](https://www.python.org/downloads/) installed on your machine
+
+### What You Will Learn
```

Comment:
Apologies for the nitpick, but for consistency, we use the contracted form for these headings ("What You'll Learn", "What You'll Need", etc)

Reply:
For some weird reason, "you'll" was making **one** section render the list below the heading as plain text with no bullets or indents. No matter what I did, it only worked if I removed the contraction. I updated all those headings to remove the contraction for consistency.